### DISCUSSION TITLE
[Flow Manager] Http protocol in Kafka broker URLs

### DISCUSSION THREAD

**Discussion by andredev93** [mia-platform/community#173] [Q&A] [answered]
Hi,

configuring the Flow Manager with Kafka as communication protocol I have to specify the Kafka broker list as in [here](https://docs.mia-platform.eu/docs/runtime_suite/flow-manager-service/configuration#kafka-example) (the broker URLs contains `http://`).

I tried a configuration similar to the example linked but I get this error: 
![Screenshot from 2023-01-18 11-04-48](https://user-images.githubusercontent.com/10067183/213142987-4c45e0c9-b2d9-4d6f-9166-d9a2681783c6.png)

But if I remove the "http/https" from the broker URLs, the Flow Manager service will work correctly.

If Flow Manager internally uses KafkaJs then I also saw that in the library [configuration](https://kafka.js.org/docs/getting-started) there is no protocol for the brokers.

Is it correct or maybe I might have wrongly configured Flow Manager somewhere else?

Thanks,
Andre

**Comment by fredmaggiowski** [comment, marked as answer]
I think this may be an issue with the documentation; however I wonder how would it work if you need `https` 🤔  @silversoul93 FYI

**Reply by frengooo** [reply]
It's an issue with the documentation (my bad, sorry). We never used the protocol too, even because Kafka uses a custom protocol if I am not wrong.

**Reply by fredmaggiowski** [reply]
Could you fix the docs please?

**Reply by andredev93** [reply]
Hi, I created a MR with the fix to the docs. Let me know in case of any problems.
